internal/pkg/test/tool/cache.go

```diff
@@ -3,13 +3,13 @@
 // LICENSE.md file distributed with the sources of this project regarding your
 // rights to use or distribute this software.
 
-package test
+package tool
```

Comment:
Please add documentation for this package. In particular it's necessary to differentiate between `github.com/sylabs/singularity/internal/pkg/test` and `github.com/sylabs/singularity/internal/pkg/test/tool` from the point of view of someone adding code to either of those packages. Think of it like this: the name "test" implies that that package contains everything related to tests, while the name "tool" is generic, so some guidance is required as to what belongs in `tool` and not in `test`.

Reply:
@mem added in https://github.com/sylabs/singularity/blob/243d57aaa91ff430b980e5f3d9ec086bcc7086eb/internal/pkg/test/tool/doc.go